tools/travis-scripts/before-install.sh

```diff
@@ -41,22 +41,6 @@ function install_python_module_for_osx()
     sudo pip install Cheetah
 }
 
-function install_latest_python()
-{
-    python -V
-    eval "$(pyenv init -)"
-    pyenv install 2.7.14
-    pyenv global 2.7.14
-    python -V
-}
```

Comment:
Can we make sure python is 2.x, not 3.x as all scripts don't support 3.x.

Reply:
Please check build output. 
python -V returns 2.7.16 for images xcode10.3, xcode11.
This code was needed because image Xcode9.3 has old version of python 2.7.10 with old networking libraries - download_deps failed

Reply:
But in future, python may be updated to v3, isn't it? I think it is better to check python version to determine if need to install python 2.7?

Reply:
On Linux/mac by default:
 ./python == 2.*  
./python3 == 3.*

Removed code was not for checking but for installing latest python 2. It is not needed any with new Xcode images